src/AppBundle/Resources/views/Restaurant/form.html.twig

```diff
@@ -164,10 +164,11 @@
   {% endif %}
 
   {% if form.contract is defined %}
-    {{ form_row(form.contract.minimumCartAmount) }}
+    {{ form_row(form.contract.minimumCartAmount, { help: 'restaurant.contract.flatDeliveryPrice.help' }) }}
     {{ form_row(form.contract.flatDeliveryPrice, { help: 'restaurant.contract.flatDeliveryPrice.help' }) }}
     {{ form_row(form.contract.customerAmount, { help: 'restaurant.contract.customerAmount.help' }) }}
     {{ form_row(form.contract.feeRate, { help: 'restaurant.contract.feeRate.help' }) }}
+    {{ form_row(form.contract.restaurantPaysStripeFee) }}
```

Comment:
Can you add an help text as well ?
Also as you suggested can you hide this field by default ?